src/settings/config.rs

```diff
@@ -105,6 +106,9 @@ impl Config {
         if let Some(title_hidden) = &self.title_hidden {
             env::set_var("NEOVIDE_TITLE_HIDDEN", title_hidden.to_string());
         }
+        if let Some(no_tabs) = &self.no_tabs {
+            env::set_var("NEOVIDE_NO_TABS", no_tabs.to_string());
```

Comment:
It's more logical if the environment variable is `NEOVIDE_TABS`, and you can set `0` and `1`. We do the same for other similar variables like `NEOVIDE_VSYNC=0/1`.

We also try to make the actual argument accept both forms `--tabs` and `--no-tabs` for example. That way you can always temporarily overwrite the environment variable or configuration file setting from the command line.

Reply:
There seems to be a little bit of inconsistency.
Some parameters have form of title-bar=on/off, another no-fork=on/off.

I feel same way about `NEOVIDE_TABS` but my idea was not to change convention already in place and just propagate option.

So, is fork=on/off, idle=on/off, tabs=on/off the way moving forward?

Reply:
Yes but I think docs still do not reflect most of it, for example:
https://github.com/neovide/neovide/blob/main/website/docs/command-line-reference.md?plain=1#L115 - no information about --no-fork vs --fork

Anyways, I changed it like you suggested. 
Let me know what you think.

BTW, I don't like the current precedence (CLI arg > config_file > ENV) where config_file has precedence over env variable.

Reply:
I agree with both of your points, the documentation could be clearer, we should fix that at some point. 

And also, that the order should be CLI > ENV > config instead, but I'm not sure how much of a breaking change that would be. I somehow missed that the order was what it is when the config file support was added.